### DISCUSSION TITLE
Problem DVC

### DISCUSSION THREAD

**Discussion by juanfe977** [vincentjzy/OpenCorr#14] [Q&A]
Hi
I have a porblem running the DVC example because i did not foud the Torus_POIs.csv file
please could you tell me were to find it? and also I would like to know if that file is from the ref_image or the tar_image
 thank you very much

**Comment by vincentjzy** [comment]
Torus_POIs.csv can be found in this repository:
https://github.com/vincentjzy/OpenCorr/tree/main/examples/dvc

The POIs are set according to the coordinate system in reference image. We put the volume images of this example on opencorr.org as the files may be a little bit too large to GitHub repository.
